lib/features/base_view/base_view.dart

```diff
@@ -6,149 +6,212 @@ import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:flutter_screenutil/flutter_screenutil.dart';
 import 'package:google_fonts/google_fonts.dart';
 
-abstract class BasePageScreen extends StatefulWidget {
-  const BasePageScreen({super.key});
+abstract class BasePage extends StatefulWidget {
```

Comment:
file name is base_view, but class name is BasePage. Should we change the same name for consistent